client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -426,6 +443,97 @@ export const useCommandsArrayWpcom = ( {
 			},
 			icon: addNewSiteIcon,
 		},
+		{
+			name: 'addNewPost',
+			label: __( 'Add new post' ),
+			searchLabel: __( 'add new post' ),
+			callback: setStateCallback( 'addNewPost' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/post/${ site.slug }` );
+				},
+			},
+			icon: postIcon,
+		},
+		{
+			name: 'addNewPostAccessSubscribers',
+			label: __( 'Add new subscribers-only post' ),
+			searchLabel: __( 'add new subscribers-only post' ),
+			callback: setStateCallback( 'addNewPostAccessSubscribers' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/post/${ site.slug }?jetpack_access=subscribers` ); // @TODO: `?jetpack_access=` requires Jetpack PR
+				},
+			},
+			icon: postIcon,
+		},
+		{
+			name: 'addNewPostAccessPaidSubscribers',
+			label: __( 'Add new paid subscribers-only post' ),
+			searchLabel: __( 'add new paid subscribers-only post' ),
+			callback: setStateCallback( 'addNewPostAccessPaidSubscribers' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/post/${ site.slug }?jetpack_access=paid-subscribers` ); // @TODO: `?jetpack_access=` requires Jetpack PR
+				},
+			},
+			icon: postIcon, // TODO: paywall icon
+		},
+		{
+			name: 'manageComments',
+			label: __( 'Manage comments' ),
+			searchLabel: __( 'manage comments' ),
+			callback: setStateCallback( 'manageComments' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/comments/${ site.slug }` );
+				},
+			},
+			icon: postCommentsIcon,
+		},
+		{
+			name: 'addSubscribers',
+			label: __( 'Add subscribers' ),
+			searchLabel: __( 'add subscribers' ),
+			callback: setStateCallback( 'addSubscribers' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/subscribers/${ site.slug }#add-subscribers` );
+				},
+			},
+			icon: subscriberIcon, // TODO: subscribers icon, currently using icon for comments
+		},
+		{
+			name: 'manageSubscribers',
+			label: __( 'Manage subscribers' ),
+			searchLabel: __( 'manage subscribers' ),
+			callback: setStateCallback( 'manageSubscribers' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/subscribers/${ site.slug }` );
+				},
+			},
+			icon: subscriberIcon, // TODO: subscribers icon, currently using icon for comments
+		},
+		{
+			name: 'downloadSubscribers',
+			label: __( 'Download subscribers as CSV' ),
+			searchLabel: __( 'download subscribers as csv' ),
+			callback: setStateCallback( 'downloadSubscribers' ),
+			siteFunctions: {
```

Comment:
Command could be filtered out if they have 0 subscribers, but I see we now filter only by info in `sites` array.

Would probably get to expensive to check for subscriber count for each site.